app/celery/scheduled_tasks.py

```diff
@@ -256,28 +256,25 @@ def switch_current_sms_provider_on_slow_delivery():
     Switch providers if there are at least two slow delivery notifications (more than four minutes)
     in the last ten minutes. Search from the time we last switched to the current provider.
     """
-    functional_test_provider_service_id = current_app.config.get('FUNCTIONAL_TEST_PROVIDER_SERVICE_ID')
-    functional_test_provider_template_id = current_app.config.get('FUNCTIONAL_TEST_PROVIDER_SMS_TEMPLATE_ID')
```

Comment:
these can now be removed from config.py entirely right

Reply:
Good find, I removed those from config and the tests now.